__tests__/test-utils-splitLines.js

```
@@ -0,0 +1,39 @@
+/* eslint-env node, browser, jasmine */
+import { splitLines } from 'isomorphic-git/internal-apis'
+
+async function collect(chunks) {
+  const fifo = splitLines(
+    (async function*() {
```

ESLint

[error] Insert `·`

(prettier/prettier)